src/views/v2/components/fields/birth.php

```diff
@@ -0,0 +1,86 @@
+<?php
+/**
+ * This template renders the Birth field.
+ *
+ * Override this template in your own theme by creating a file at:
+ * [your-theme]/tribe/tickets/v2/components/fields/birth.php
+ *
+ * @TODO: Update this file once this one is merged https://github.com/moderntribe/event-tickets/pull/1677
+ *
+ * @since   TBD
+ *
+ * @version TBD
+ *
+ * @see     Tribe__Tickets_Plus__Meta__Field__Birth
+ */
+
+$field_name  = tribe_tickets_ar_field_name( $ticket->ID, $field->slug );
+$field_id    = tribe_tickets_ar_field_id( $ticket->ID, $field->slug );
+$required    = tribe_tickets_ar_field_is_required( $field );
+$birth_field = $field;
+$field       = (array) $field;
+$disabled    = false;
+$classes     = [
+	'tribe-common-b1',
+	'tribe-tickets__form-field',
+	'tribe-tickets__form-field--required' => $required,
+];
+?>
+<div class="tribe_horizontal_datepicker__container">
+	<div <?php tribe_classes( $classes ); ?>>
+		<label
+			class="tribe-common-b2--min-medium tribe-tickets__form-field-label"
+			for="<?php echo esc_attr( $birth_field->month_id ); ?>"
```

Comment:
Is this the bugged line we fixed in G20.06 PRs?

Reply:
Might be! I grabbed the files from what we had a couple weeks, and updated a bit the CSS classes and added template tags to avoid repeating ourselves when setting the field ID and name.